server/pulp/plugins/loader/manager.py

```diff
@@ -65,8 +63,6 @@ def _load_unit_models(self):
 
             # Attach all the signals
             model_class.attach_signals()
-            signals.post_init.connect(model_class.post_init_signal, sender=model_class)
```

Comment:
I cleaned these out since the attach_signals() method above performs these actions